haystack/utils/deepsetcloud.py

```diff
@@ -967,3 +1112,169 @@ def get_file_client(
         """
         client = DeepsetCloudClient(api_key=api_key, api_endpoint=api_endpoint)
         return FileClient(client=client, workspace=workspace)
+
+
+class DeepsetCloudExperiments:
+    """
+    A facade to conduct and manage experiments within deepset Cloud.
+
+    To start a new experiment run:
+    1. Choose a pipeline to evaluate using `list_pipelines()`
+    2. Choose an evaluation set using `list_evaluation_sets()`
+    3. Create and start a new run using `create_and_start_run()`
+    4. Track the run using `get_run()`
+    """
+
+    @classmethod
+    def list_pipelines(
+        cls, workspace: str = "default", api_key: Optional[str] = None, api_endpoint: Optional[str] = None
+    ) -> List[dict]:
+        """
+        Lists all pipeline configs available on Deepset Cloud.
```

Comment:
Only a few of the newly implemented methods have docstrings yet. Let's add docstrings where they are missing.

Reply:
can we say "configurations" instead of "configs"? I know it's nitpicking.. ;)

Reply:
and can we change "Deepset Cloud" to "deepset Cloud"

Reply:
I'll keep it "configs" as "pipeline config" is something haystack users are familiar with.